javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+var fullName = 'Henry Nichols';
+var isCool = false;
+var totalPets = 1;
+var unknown = null;
+var never =
```

Comment:
If you are assigning an empty value to a variable you are declaring, the syntax should be like this: `var never;`